metacat-metadata-mysql/src/test/resources/sql/prepare-test.sql

```diff
@@ -36,6 +36,20 @@ CREATE TABLE metacat.definition_metadata (
   UNIQUE KEY name (name)
 ) DEFAULT CHARSET=latin1;
 
+-- Create syntax for TABLE 'definition_metadata'
+CREATE TABLE metacat.partition_definition_metadata (
+  id bigint(20) NOT NULL AUTO_INCREMENT,
+  version bigint(20) NOT NULL,
+  created_by varchar(255) NOT NULL,
+  data longtext NOT NULL,
+  date_created datetime NOT NULL,
```

Comment:
if you want greater granularity you can set `datetime(3)` for milliseconds... not sure if you want that or not but that's what is done in Genie